### PR TITLE
Adds LilyGo T Relay S3 board definition

### PR DESCRIPTION
**Pull request by LordRafa**
*By completing this PR sufficiently, you help us to review this Pull Request quicker and also help improve the quality of Release Notes*

## Description of Change
Add  [LilyGo T Relay S3](https://github.com/Xinyuan-LilyGO/LilyGo-T-Relay/blob/main/docs/RELAY_ESP32S3.MD) board definition

## Tests scenarios
Tested compilation against basic examples.

## Related links

Pins documentation is wrong. I check pins avaiale directly on the [schematic](https://github.com/Xinyuan-LilyGO/LilyGo-T-Relay/blob/main/Schematic/T_Relay6_ESP32S3.pdf)

### CODE REVIEW

**Comment by P-R-O-C-H-Y** [comment]
@LordRafa can you please sign CLA?

File: variants/lilygo_t_relay_s3/pins_arduino.h [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+#define USB_VID 0x303a
+#define USB_PID 0x1001
+
+#define EXTERNAL_NUM_INTERRUPTS 46
+#define NUM_DIGITAL_PINS        48
+#define NUM_ANALOG_INPUTS       20
+
+#define analogInputToDigitalPin(p)  (((p)<20)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<48)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 46)
```
Comment (P-R-O-C-H-Y) [inline variants/lilygo_t_relay_s3/pins_arduino.h:16]: This part have been moved and its defined globally per SoC. Please remove it.
```suggestion
```

**Review by P-R-O-C-H-Y — CHANGES_REQUESTED** [review]
@LordRafa PTAL

**Review by lucasssvaz — CHANGES_REQUESTED** [review]
Please also recommit your changes using only your Github username otherwise the CLA won't be able to be properly signed

**Comment by me-no-dev** [comment]
Closing for lack of response. We can not merge PRs without signed CLA. @LordRafa feel free to open a new PR if you are willing to sign the CLA

**Pull request closed without merging**